apps/nextjs-website/src/app/page.tsx

```diff
@@ -63,13 +57,11 @@ const Home = async () => {
           svgPath: product.svgPath,
         }))}
       />
-      {nextWebinars.length !== 0 && (
-        <WebinarsSection
-          title={homepage.webinarsSection.title}
-          description={homepage.webinarsSection.description}
-          webinars={[...nextWebinars]}
-        />
-      )}
+      <WebinarsSection
```

Comment:
We need to understand whether the section (WebinarSection) should remain or disappear from the home page when the webinar ends (IMHO it should remain since for now there's no webinar list page) and if for now, the section does not change it is useless to filter by date.

Reply:
If the filter by date is necessary, I think that the component should be rendered `dynamic` like `<NotSsrWebinarHeaderBanner webinars={webinars} />` to avoid the "cookie banner bug"